.github/workflows/release-asset-audit.yml

```diff
@@ -0,0 +1,45 @@
+name: Release Asset Audit
+
+on:
+  workflow_dispatch:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    # Run once an hour
+    - cron:  '5 * * * *'
```

Comment:
There is no event to trigger the workflow on upload? I am a bit worried about leaving potential vectors up for in the worst case an hour.

Reply:
That's a good point. Looks like `release.edited` might be what we want here. The documentation seems to be a bit sparse though.

Reply:
I've added a release trigger now, but it doesn't seem to work.